tools/run_tests/run_xds_tests.py

```diff
@@ -31,10 +32,20 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2_grpc
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger()
 console_handler = logging.StreamHandler()
 logger.addHandler(console_handler)
 
+
+def parse_port_range(port_arg):
```

Comment:
![](https://media2.giphy.com/media/h1zJMhT5XOT927e0aw/giphy.gif?cid=790b7611aeaba6ee6058b884b78866a585136d16c8c65a0b&rid=giphy.gif)

Reply:
Is there a better way to do this? Didn't really want to introduce two new flags (so `--*_port` along with `--port_min` and `--port_max`, as then the user has to know which combination takes precedence)